packages/notifier/src/notifier.js

```diff
@@ -92,7 +92,7 @@ export const produceNotifier = (initialState = undefined) => {
 
     currentResponse = harden({
       value: finalState,
-      updateHandle: undefined,
+      updateHandle: {},
```

Comment:
When the state has been resolved, there should not be a new updateHandle.
Yes, that means that the declaration on line 15 should change to `| undefined`.

Reply:
Got it.  Thanks!